src/explore-education-statistics-admin/src/pages/release/data/ReleaseAncillaryFileReplacePage.tsx

```diff
@@ -8,28 +8,30 @@ import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
 import useFormSubmit from '@common/hooks/useFormSubmit';
 import React from 'react';
 import Link from '@admin/components/Link';
-import { generatePath, RouteComponentProps } from 'react-router';
+import { generatePath } from 'react-router';
 import LoadingSpinner from '@common/components/LoadingSpinner';
 import Yup from '@common/validation/yup';
 import { Formik } from 'formik';
 import { Form } from '@common/components/form';
 import Button from '@common/components/Button';
 import releaseAncillaryFileService from '@admin/services/releaseAncillaryFileService';
 import FormFieldFileInput from '@common/components/form/FormFieldFileInput';
-import ButtonText from '@common/components/ButtonText';
-import FormattedDate from '@common/components/FormattedDate';
 import AncillaryFileDetailsTable from '@admin/pages/release/data/components/AncillaryFileDetailsTable';
+import { useHistory, useParams } from 'react-router-dom';
 
 interface FormValues {
   file: File | null;
 }
 
-const ReleaseAncillaryFileReplacePage = ({
-  history,
-  match: {
-    params: { publicationId, releaseId, fileId },
-  },
-}: RouteComponentProps<ReleaseAncillaryFileReplaceRouteParams>) => {
+const ReleaseAncillaryFileReplacePage = () => {
+  const history = useHistory();
+
+  const { publicationId, releaseId, fileId } = useParams<{
```

Comment:
We can just use the `ReleaseAncillaryFileReplaceRouteParams` interface here instead of having to specify a new type